network/p2p/libp2pNode.go

```diff
@@ -45,6 +46,10 @@ const (
 
 	// maximum number of milliseconds to wait between attempts for a 1-1 direct connection
 	maxConnectAttemptSleepDuration = 5
+
+	// timeout for FindPeer queries to the DHT
+	// TODO: is this a sensible value?
+	findPeerQueryTimeout = 10 * time.Second
```

Comment:
We're already tracking this in essentially #1184 . I think this value is a good start, but we may want to rename the issue @smnzhu